WooCommerce/Classes/ViewRelated/Orders/Order Creation/EditableOrderViewModel.swift

```diff
@@ -179,7 +179,7 @@ final class EditableOrderViewModel: ObservableObject {
             storageManager: storageManager,
             stores: stores,
             supportsMultipleSelection: isProductMultiSelectionBetaFeatureEnabled,
-            isClearSelectionEnabled: false,
+            isClearSelectionEnabled: isProductMultiSelectionBetaFeatureEnabled,
```

Comment:
❓ Could you explain again why we have this config on both the view model and the configuration for the selector? Are we planning to unify them?

Reply:
Yes, at the moment is temporary due keeping Single-Selection, as this uses the config as well but does not have a "Clear Selection" button. Once Multi-Selection is done, and we remove the flags we can also get rid of this extra bit as there will be only one value for both products and variations for any config.